client/app/bundles/course/assessment/submission/components/GetHelpChatPage/ChatInputArea.tsx

```diff
@@ -51,6 +55,8 @@ const ChatInputArea: FC<ChatInputAreaProps> = (props) => {
     isResetting ||
     isRequestingLiveFeedback ||
     isPollingLiveFeedback ||
+    !currentThreadId ||
+    syncStatus === GET_HELP_SYNC_STATUS.Failed ||
```

Comment:
what about `isThreadExpired`? Do we still allow them to type?

Reply:
We decided not to allow the user to type when `isThreadExpired` is true